packages/client/hmi-client/src/services/project.ts

```diff
@@ -195,6 +196,17 @@ function iconClassname(type: ProjectAssetTypes | string | null): string {
 	return 'pi pi-circle';
 }
 
+/**
+ * Get the xdd_uri of a Project Document
+ */
+function getDocumentAssetXddUri(project: IProject, assetId: Tab['assetId']): string | null {
```

Comment:
Where would we expect assetId to be null?

Reply:
For new assets, when creating them on the front end.
Or just simply when the document.vue first mount and the props aren't filled yet.